packages/build-tools/src/steps/functions/installMaestro.ts

```diff
@@ -1,7 +1,8 @@
 import assert from 'assert';
+import fs from 'fs';
```

Comment:
I think we usually use `fs-extra` in other parts of the code. Maybe it's worth using it here as well?

Reply:
So in fact I moved this away from using `fs-extra`. If we can not use it, why use it?

Reply:
Oh, sorry, I see 🙈

Reply:
I mean, this may be a wrong decision and maybe we should decide to stick to `fs-extra`. I'm not sure though — for instance `www` only uses `fs`. Wanted to experiment and see if it works!